packages/neo3-invoker/README.MD

```diff
@@ -10,9 +10,211 @@
 </p>
 
 # Neo3-Invoker
+
 Neo3-Invoker is a specification of how SmartContract client SDKs can interact with different invoker libraries such Neon-JS or WalletConnect.
 
 With Neo3-Invoker a SDK don't need to reimplement the wheel and can focus on the SmartContract communication format.
 Taking advantage of the declarative nature of the specification, the SDK can be used with different invoker libraries, it only depends on what the dApp developer wants to use.
 
 Visit the [main page](../../README.md) of the project.
+
+## Arguments examples:
+
+For these examples, we are using the [NeonInvoker](../neon-invoker/README.md) that is an implementation of Neo3Invoker, so we are starting from the point that NeonInvoker is already initialized.
```

Comment:
Actually, we are not using neon-invoker. It seems these examples below could be for any neo3-invoker. Right?

Reply:
It is true. I will change to: "For these examples, we are starting from the point where an invoker has already been initialized.". What do you think ?

Reply:
Good